pgmpy/models/BayesianModel.py

```diff
@@ -315,10 +315,12 @@ def remove_cpds(self, *cpds):
                 cpd = self.get_cpds(cpd)
             self.cpds.remove(cpd)
 
-    def get_cardinality(self, node):
+    def get_cardinality(self, node=None):
```

Comment:
@yashu-seth But then I think it will be of no use to take a variable node. Both methods will be like
>>>get_cardinality()['node']
>>>get_cardinality('node')['node']